src/main/java/uk/gov/dwp/dataworks/provider/KMSDataKeyGeneratorProvider.java

```diff
@@ -12,10 +13,11 @@
 import java.util.Base64;
 
 @Service
+@Profile("KMS")
```

Comment:
What does turning on this profile do? Is it a Spring Boot profile that turns something on/enables something?

Reply:
As discussed